backend/core/test/factories/listing.ts

```diff
@@ -0,0 +1,79 @@
+import { Factory } from "fishery"
+import { nanoid } from "nanoid"
+import { Listing } from "../../src/entity/Listing"
+
+export default Factory.define<Listing>(({ sequence, factories }) => ({
+  id: nanoid(),
+  preferences: [],
+  units: [factories.unit.build()],
+  attachments: [],
+  acceptingApplicationsAtLeasingAgent: true,
+  acceptingApplicationsByPoBox: true,
+  acceptingOnlineApplications: false,
+  acceptsPostmarkedApplications: true,
+  accessibility: "There is a total of 5 ADA units in the complex",
+  amenities: "Community Room, Laundry Room",
+  applicationDueDate: "2019-12-31T15:22:57.000-07:00",
+  applicationOpenDate: null,
+  applicationFee: "30.0",
+  applicationOrganization: `Triton-${sequence}`,
+  applicationAddress: {
+    city: `Foster City-${sequence}`,
+    street: "55 Triton Park Lane",
+    zipCode: "94404",
+    state: "CA",
+    county: "San Jose",
```

Comment:
Do we want to use [faker](https://github.com/marak/Faker.js/) for making more varied data here? It has address utilities...

If determinism is an issue, faker can de made deterministically random using `faker.seed(deterministicSeed)` (so the random data it generates will be the same with the same seed value)

Reply:
I agree faker could be useful here, perhaps in combination with a set of static test data that we know will be consistent for human QA. I think it can wait for a later PR, though.